.github/workflows/run-tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           run: |
             composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
             composer require "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-            composer update --prefer-dist --no-interaction --no-progress
+            composer update --prefer-dist --no-interaction --no-progress --no-plugins
```

Comment:
Why is this needed?

Reply:
This is down to Composer 2 (the PHP dependancy management tool) prohibiting the automatic execution of library scripts unless explicitly allowed - they're like lifecycle hooks that are fired during dependancy installation. 

Can you see any reason this works on 0.4 but not 0.0.x @jimmy718?